main.py

```diff
@@ -0,0 +1,40 @@
+import requests
+from urllib.parse import urlparse 
+
+Apache_Type = ["text/plain", "text/plain; charset=ISO-8859-1", 
+                "text/plain; charset=iso-8859-1", 
+                "text/plain; charset=UTF-8"]
+
+check_for_apache = False
```

Comment:
Should this be defined within `_get_resource_metadata`?

Reply:
Mutable objects should not be defined at the module level most of the time, they should be defined within the function that uses them, and be passed around as function parameters if needed.